docs/2.10.0/docs/canton/usermanual/pruning.rst

```diff
@@ -177,6 +177,9 @@ Current Limitations
   it shares a contract. If a participant becomes defunct and stops sending commitments, pruning of the participant will
   not work. Therefore, before you disable a participant, please make sure that is not involved in any contract. This is
   a current limitation.
+- Participants do not allow pruning of state more recent than the most recently observed transaction minus the
+  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all state
+  older than the configured retention period.
```

Comment:
```suggestion
  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all states
  older than the configured retention period.
```

Reply:
Thanks Carrie. To use the plural form, I am inclined to change from `state` or `states` to `archived contracts and transactions`. I will make the changes.